x-pack/plugins/fleet/common/types/models/agent_policy.ts

```diff
@@ -39,6 +39,7 @@ export interface AgentPolicy extends Omit<NewAgentPolicy, 'id'> {
   updated_at: string;
   updated_by: string;
   revision: number;
+  agents?: number;
```

Comment:
agents field is present in agent policies, but it seems to be missing from type. now sure why.